ProjFS.Mac/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -250,7 +232,7 @@ static VirtualizationRootHandle FindOrDetectRootAtVnode(vnode_t _Nonnull vnode,
 
 static VirtualizationRootHandle FindUnusedIndex_Locked()
 {
-    for (VirtualizationRootHandle i = 0; i < s_maxVirtualizationRoots; ++i)
+    for (uint32_t i = 0; i < s_maxVirtualizationRoots; ++i)
```

Comment:
:-) We were using a handle as int?

Reply:
Well, it's typedef’d to an `int16_t`. But yeah, turns out, not a good idea.